src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer {
```

Comment:
```suggestion
public class ExcavatorProducer implements MachineProducer< your parameterization >{
```
let's not use raw type